app/services/doc_auth/lexis_nexis/config.rb

```diff
@@ -5,8 +5,6 @@ module LexisNexis
       :base_url, # required
       :request_mode,
       :trueid_account_id,
-      :trueid_liveness_cropping_workflow,
```

Comment:
these configs are still needed for acuant, should they be renamed since we took liveness out?